pkg/pipeline/store_test.go

```diff
@@ -124,13 +128,13 @@ func TestConfigStore_Delete(t *testing.T) {
 	want := &Instance{ID: uuid.NewString()}
 
 	err := s.Set(ctx, want.ID, want)
-	assert.Ok(t, err)
+	is.NoErr(err)
 
 	err = s.Delete(ctx, want.ID)
-	assert.Ok(t, err)
+	is.NoErr(err)
 
 	got, err := s.Get(ctx, want.ID)
-	assert.Error(t, err)
-	assert.True(t, cerrors.Is(err, database.ErrKeyNotExist), "expected error for non-existing key")
-	assert.Nil(t, got)
+	is.True(err != nil)
+	is.True(cerrors.Is(err, database.ErrKeyNotExist))
```

Comment:
you can add the reason as an inline comment, which will be logged when this statement fails
```suggestion
	is.True(cerrors.Is(err, database.ErrKeyNotExist)) // expected error for non-existing key
```

Reply:
also, please solve the conflicts so the workflows and tests could run

Reply:
when running the test, should that sepecific test fail?